quinn-proto/src/packet.rs

```diff
@@ -654,15 +661,27 @@ impl PlainHeader {
     }
 }
 
+/// A Plain QUIC Header
 #[derive(Clone, Debug)]
-pub(crate) struct PlainInitialHeader {
+pub struct PlainInitialHeader {
```

Comment:
If all the other enum variants' fields are public, these probably should be too, and then the accessors won't be needed.